beacon-chain/core/helpers/committee.go

```diff
@@ -385,25 +374,16 @@ func ShuffledIndices(state *stateTrie.BeaconState, epoch uint64) ([]uint64, erro
 		return nil, errors.Wrapf(err, "could not get seed for epoch %d", epoch)
 	}
 
-	vals := state.Validators()
-	indices := make([]uint64, 0, len(vals))
-	for i, v := range vals {
-		if IsActiveValidator(v, epoch) {
-			indices = append(indices, uint64(i))
+	indices := make([]uint64, 0, state.NumofValidators())
+	// ignore error as no error is returned in above callback.
```

Comment:
Callback above? Is the comment necessary?

Reply:
that is outdated, removing it